assembl/views/__init__.py

```diff
@@ -140,20 +216,33 @@ def get_bundle_css_hash(href):
         return get_resource_hash(r'/build/bundle\.(.*)\.css$', href)
 
     def get_theme_css_hash(href):
-        return get_resource_hash(r'/build/theme_' + re.escape(theme_name) + r'_web\.(.*)\.css$', href)
+        if bucket_name == 'default' or theme_name == 'default':
+            regex = r'/build/themes/default/theme_' + re.escape(theme_name) + r'_web\.(.*)\.css$'
+        else:
+            regex = r'/build/themes/' + re.escape(bucket_name) + '/[^/]+/theme_' + re.escape(theme_name) + r'_web\.(.*)\.css$'
+        return get_resource_hash(regex, href)
+
+    def get_theme_js_hash(href):
+        if bucket_name == 'default' or theme_name == 'default':
+            regex = r'/build/themes/default/theme_' + re.escape(theme_name) + r'_web\.(.*)\.js$'
+        else:
+            regex = r'/build/themes/' + re.escape(bucket_name) + '/[^/]+/theme_' + re.escape(theme_name) + r'_web\.(.*)\.js$'
+        return get_resource_hash(regex, href)
 
     soup = BeautifulSoup(source)
     bundle = soup.find(src=get_bundle_hash)
     bundle_css = soup.find(href=get_bundle_css_hash)
     theme_css = soup.find(href=get_theme_css_hash)
+    theme_js = soup.find(src=get_theme_js_hash)
```

Comment:
You don't have theme_js in the jinja template in production. This is only for development.

Reply:
Yeah, it's right. Copy/pasta that I didn't remove. But just to be clear, the webpack-dev-server only serves the js file as is, with no hash, right?

Reply:
Yes, that's right.